src/executables/csql.c

```diff
@@ -1049,7 +1049,8 @@ csql_do_session_cmd (char *line_read, CSQL_ARGUMENT * csql_arg)
 	{
 	  if (csql_arg->sysadm && au_is_dba_group_member (Au_user))
 	    {
-	      au_disable ();
+	      int dummy;
+	      AU_DISABLE (dummy);
```

Comment:
```suggestion
	      Au_disable = true;
```
dummy is not used after this line and can be removed.

Reply:
To hide Au_disable, I prefer to call `AU_DISABLE (value)`.